CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
   - key `data`.`layman`.`last-migration` was removed from [GET Version](doc/rest.md#get-version) response
   - body parameter `sld` was removed from [POST Workspace Publications](doc/rest.md#post-workspace-layers) and [PATCH Workspace Publication](doc/rest.md#patch-workspace-layer)
   - workspace-related endpoints which did not include `/workspaces` in their path were removed 
+- [#701](https://github.com/LayerManager/layman/pull/701) Check bounding bbox of normalized raster before posting to GeoServer.
```

Comment:
Add something like "and stop to check that Layer is available in GetCapabilities after publishing to GeoServer, Layman"

Reply:
Implemented